mne/decoding/time_gen.py

```diff
@@ -665,26 +742,28 @@ def _sliding_window(times, window_params, sfreq):
         def find_time_idx(t):  # find closest time point
             return np.argmin(np.abs(np.asarray(times) - t))
 
-        start = find_time_idx(window_params['start'])
-        stop = find_time_idx(window_params['stop'])
-        step = int(round(window_params['step'] * sfreq))
-        length = int(round(window_params['length'] * sfreq))
+        start = find_time_idx(window['start'])
+        stop = find_time_idx(window['stop'])
+        step = int(round(window['step'] * sfreq))
+        length = int(round(window['length'] * sfreq))
 
         # For each training slice, give time samples to be included
         time_pick = [range(start, start + length)]
         while (time_pick[-1][0] + step) <= (stop - length + 1):
             start = time_pick[-1][0] + step
             time_pick.append(range(start, start + length))
-        window_params['slices'] = time_pick
+        window['slices'] = time_pick
+    window['times'] = _set_window_time(window['slices'], times)
+    return window
 
-    # Keep last training times in milliseconds
-    t_inds_ = [t[-1] for t in window_params['slices']]
-    window_params['times'] = times[t_inds_]
 
-    return window_params
+def _set_window_time(slices, times):
+    """Defines time as the last training time point"""
+    t_inds_ = [t[-1] for t in slices]
```

Comment:
`inds_` ? Please make it have `idx` in the name if it is to be used for indexing